src/repart/meson.build

```diff
@@ -49,13 +41,13 @@ executables += [
 ]
 
 if conf.get('ENABLE_REPART') == 1
```

Comment:
The check can be inverted (check ENABLE_REPART at the beginning of file, if disabled exit early with subdir_done())

Reply:
It's just a few lines… I don't see a particular benefit in this case.

Reply:
I like the suggestion by @YHNdnzj, but not a big matter here. Let's merge this as is. @YHNdnzj If you'd like, please create a follow-up PR.